tests/python/pants_test/engine/test_fs.py

```diff
@@ -134,7 +143,7 @@ def test_walk_parent_link(self):
 
     def test_walk_escaping_symlink(self):
         link = "subdir/escaping"
-        dest = "../../this-is-probably-nonexistent"
+        dest = "../../"
```

Comment:
What happened here?

Reply:
The test failed because we started propagating errors from notify, in this case that the path it was trying to watch didn't exist.

Reply:
The dest path still tries to link outside of the build root so the test should be the same.